common/bytes.go

```diff
@@ -107,14 +107,9 @@ func ReadVarInt(buff []byte) (ret uint64) {
 	return
 }
 
-// Copy bytes
-//
 // Returns an exact copy of the provided bytes
 func CopyBytes(b []byte) (copiedBytes []byte) {
-	copiedBytes = make([]byte, len(b))
-	copy(copiedBytes, b)
-
-	return
+	return append([]byte{}, b...)
```

Comment:
k. commit dae698cd6d0ec01c9dee61a3cf822f494725b542 is standalone and if it is used a lot (?) then note the current version is doing 3x the work. You may want to cherry pick that.